gains:
  Katrr: 30
  Ktask_trans: 12
  Dtask_trans: 12
  Ktask_orien: 12
  Dtask_orien: 12
  Kjoint: 12
  Djoint: 12
